src/ros_license_toolkit/common.py

```diff
@@ -16,7 +16,10 @@
 
 """Common utility functions."""
 
-from typing import Any, Dict, Optional
+import json
+from typing import Any, Dict, List, Optional
+
+REQUIRED_PERCENTAGE_OF_LICENSE_TEXT = 95.0
 
 REQUIRED_PERCENTAGE_OF_LICENSE_TEXT = 95.0
```

Comment:
```suggestion
REQUIRED_PERCENTAGE_OF_LICENSE_TEXT = 95.0
```
Why is this here twice?